### DISCUSSION TITLE
How to reduce spam emails from github?

### DISCUSSION THREAD

**Discussion by morkeltry** [community/community#146511] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have already filed a support ticket asking how to turn off the notification that gets emailed to me every time I login.
Github tells me I can't, I just have to deal with it.

I am now wondering whether I can turn off the multiple notifications I get when each access token is set to expire.
Like, I know it's about to expire. I set the expiry date, and I set it at what it is for a reason.

Is this another 'force it down the users throat until they turn off all the security options' kind of logic, that it takes the org (github) five years of forcing on users before some thought leader explains, as their users have been doing for years, why it is counterproductive?

Anyway, I would like to be able to turn it off, or reduce the frequency and number of these unwanted emails.
Alternatively, does anyone know a clever hack so that useless notifications can be redirected to, eg, a specific email address just for spam, without the threat of lockout that comes with that spam email being one of your primary emails?
Or any other similar hack?

**Comment by imtiazvive** [comment]
I understand your frustration with the notifications. Unfortunately, GitHub doesn't offer an option to turn off these expiration reminders. However, one workaround might be using email filters or rules in your email client to automatically move these notifications to a separate folder or redirect them to a "**spam**" email address, without affecting your primary one.

For instance, if you’re using Gmail, you can set up filters that catch emails with certain subject lines or keywords like **"access token"** and redirect them accordingly.

It would be helpful if GitHub offered more flexibility in managing these notifications, especially for users who are intentionally managing their own access tokens. Hopefully, they’ll take user feedback into account in the future.
